images/cfw/usr/etc/system/webhmi/templates/index.html

```diff
@@ -0,0 +1,163 @@
+    <style>
+        body {
+            background-color: #f0f0f0; /* Light grey background */
+            font-family: 'Arial', sans-serif;
+            margin: 0;
+            padding: 0;
+        }
+        .container {
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            align-items: center;
+            min-height: 100vh;
+            padding: 20px;
+        }
+        header, footer {
+            width: 100%;
+            background-color: #333;
+            color: white;
+            text-align: center;
+            padding: 3px 0;
+        }
+        .content {
+            background: white;
+            padding: 20px;
+            border-radius: 8px;
+            box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+            width: 100%;
+            max-width: 800px;
+        }
+        .centered-content {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            gap: 20px;
+        }
+        .image-container {
+            display: flex;
+            justify-content: center;
+            gap: 20px;
+            flex-wrap: wrap;
+        }
+        .image-container img {
+            border-radius: 8px;
+            box-shadow: 0 0 5px rgba(0, 0, 0, 0.1);
+        }
+        .buttons {
+            display: flex;
+            justify-content: center;
+            gap: 10px;
+            flex-wrap: wrap;
+        }
+        .buttons button, .buttons form button {
+            font-size: 16px;
+            padding: 10px 20px;
+            border: none;
+            border-radius: 5px;
+            cursor: pointer;
+            transition: background-color 0.3s;
+        }
+        .buttons button:hover, .buttons form button:hover {
+            background-color: #555;
+            color: white;
+        }
+        .dark-mode {
+            background-color: #333;
+            color: white;
+        }
+        .dark-mode header, .dark-mode footer {
+            background-color: #222;
+        }
+        .popup {
+            position: fixed;
+            top: 20px;
+            right: 20px;
+            background-color: #333;
+            color: white;
+            padding: 10px 20px;
+            border-radius: 5px;
+            box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+            display: none;
+        }
+    </style>
+    <script>
+        function refreshImage() {
+            var img = document.getElementById('liveImage');
+            img.src = '/current_image?' + new Date().getTime();
+        }
+        setInterval(refreshImage, 5000); // Refresh every 5000 milliseconds (5 seconds)
+
+        function refreshModelImage() {
+            var img = document.getElementById('modelImage');
+            img.src = '/current_model_image?' + new Date().getTime();
+        }
+        setInterval(refreshModelImage, 180000); // Refresh every 180000 milliseconds (3 minutes)
```

Comment:
yeah this is a primo example of why one might want to use a websocket to push updates :)